operator/templates/statefulset.yaml.j2

```diff
@@ -156,3 +156,23 @@ spec:
         - mountPath: {{ geoip['mountPath'] }}
           name: geoip
       {% endif %}
+
+      {% if metrics is defined %}
```

Comment:
Also here the sidecar container needs different vars and mounts depending on the sidecar ref: https://github.com/kuskoman/logstash-exporter